docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,98 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-automatically-howto"]
+= Update projects to Quarkus 3 automatically
+include::_attributes.adoc[]
+:categories: core
+:extension-status: "experimental"
+
+Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+include::{includes}/extension-status.adoc[]
+
+== Introduction
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running an update command.
+The update command uses OpenRewrite recipes to automatically update most of your project's dependencies, source code, and documentation.
+
+// Give the link to the OpenRewrite recipe?
+// How does the user know what the update command does or does not cover?
+// In the migration guide, can we identify which items are/are not covered by the OpenRewrite recipe?
+// Do we want to invite feedback? For example: If it does not work for you, we could have missed something, or you are using extensions not yet supporting Quarkus 3.
+// https://github.com/quarkusio/quarkus/issues[Let us know] in either case so we can improve the migration script together.
```

Comment:
Any thoughts on these questions?

Reply:
> Give the link to the OpenRewrite recipe?

We could but it's hard to understand what it is doing at a glance as it's very long.

> In the migration guide, can we identify which items are/are not covered by the OpenRewrite recipe?

We hope to do that at some point but we are not there yet.

> Do we want to invite feedback?

Yes, please.

Reply:
Updated. Thanks.